### PR TITLE
Lowers romerol cost to 20 TC

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
tweak: Lowered romerol cost to 20 TC
/:cl:

braaaaaaaaaains...

### CODE REVIEW

**Comment by tortellinitony** [comment]
no

**Comment by Arzion5** [comment]
Yes

**Comment by PopNotes** [comment]
build this

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -667,8 +667,8 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 /datum/uplink_item/stealthy_weapons/romerol_kit
 	name = "Romerol"
 	desc = "A highly experimental bioterror agent which creates dormant nodules to be etched into the grey matter of the brain. On death, these nodules take control of the dead body, causing limited revivification, along with slurred speech, aggression, and the ability to infect others with this agent."
-	item = /obj/item/storage/box/syndie_kit/romerol
-	cost = 25
+	item = /obj/item/weapon/storage/box/syndie_kit/romerol
```
Comment (GuyonBroadway) [inline code/modules/uplink/uplink_item.dm]: should be /obj/item/storage/box/syndie_kit/romerol

delete the /weapon

filepaths got refacotred as /weapon didn;t actually do anything.

Comment (PopNotes) [reply code/modules/uplink/uplink_item.dm]: o

**Comment by GuyonBroadway** [comment]
also this can be made modular by adding

`/datum/uplink_item/stealthy_weapons/romerol_kit
     cost = 20`
into our uplink_item.dm in the hippie folder.

**Comment by GuyonBroadway** [comment]
with the cost under and tabbed in said file of course. Minor change yes but equally minor modularization. Less conflicts we get the better.

**Comment by GuyonBroadway** [comment]
Codes good.  Balance wise zombies are shit. You can stun and cuff them.  You can ARREST zombies!!.

**Comment by GuyonBroadway** [comment]
Which is to say I think romerol is gonna be fine gameplay wise.

**Comment by McBawbaggings** [comment]
nah, i like romerol being a surplus only item, just takes 1 traitor every round to make zombies which would tire out the idea

**Comment by GuyonBroadway** [comment]
The thing is /tg/ in their quest for _infinite fun_ removed its surplus and discount chances. If we just added those back in so you get a romerol discount once in a while, life would be more amusing.

**Comment by McBawbaggings** [comment]
>removed its surplus chance
wait hold the fuck up

how do you get it then?

**Comment by GuyonBroadway** [comment]
_Jolly co-operation with other traitors_.

For more information read this wiki article https://tgstation13.org/wiki/Byond_the_impossible

**Comment by JamieH** [comment]
there is a reason this is so hard to get.

**Comment by banthebantz** [comment]
I fear every round is going to be shit up by romerol smoke and foam grenades

**Comment by McBawbaggings** [comment]
you should still be able to get this in surplus, it's set to 100.

**Comment by GuyonBroadway** [comment]
Code is ok,. Still needs the head approval.

**Comment by Chnkr** [comment]
- not modular
- heads disagree
- shitty change

👎 PR closed, feel free to reopen and/or create a new one

**Comment by GuyonBroadway** [comment]
It was modular you dingus!

**Comment by PopNotes** [comment]
Hey shitass, it **is** modular and a total of one head gave their opinion on the PR.

Reopen this.

**Comment by GuyonBroadway** [comment]
yeah imma open this till at least two heads give a solid -1

**Comment by Pyko1** [comment]
Didn't we have alien parasite for traitor miner at one point, that we had to remove since it always ruined the round for other antags since, it  was always ALIUMS!

Would not this be the exactly same thing except ZOMBEES!

**Comment by PopNotes** [comment]
aliums are a hell of a lot more important than zambies, which are fairly easy to kill and don't even have stun immunity

**Comment by PopNotes** [comment]
merge dis

**Comment by GuyonBroadway** [comment]
not without a +2, bitch to admins more. A successful code review does not count as a +1.

**Comment by PopNotes** [comment]
foogck

**Comment by lcass** [comment]
Nah, if you really want to see zombies every single round this is how you do it. Seriously chemist traitor gets romerol, makes a bunch of smoke grenades. Bam entire sec and command staff are zombies in a normal round.

Readd it to surplus and discounts if you want it back but doing this is just asking for trouble.

**Comment by Arzion5** [comment]
I'd like to see romerol actually being used. Dealing with zombies is easy as long as you carry a sharp weapon to decapitate them with.

**Comment by PopNotes** [comment]
@lcass Do you see holoparasites every single round?
No, no you don't, and holoparasites are way more useful/entertaining than romerol.

This is a shit argument, stop using it.

**Comment by JamieH** [comment]
/tg/ actually had a huge issue with romerol being used every round, hence the difficulties in getting it.

**Comment by PopNotes** [comment]
@JamieH 
One, that's /tg/, of course they would have difficulty with it. We're the robusting server.
Two, traitor is a much rarer gamemode here in general because of roundtype chance balancing.

**Comment by bardofrage** [comment]
Idk, it's fine as a surplus item. Zombies should be a rare occurrence, not something that happens every time we roll traitors.

**Comment by DinduStuffin** [comment]
I'm with bard on this one. I feel like it'd result in constant zombie spam with almost any traitor round.

**Comment by doobiedoo23** [comment]
fine as a surplus item

don't like this PR at all

**Comment by JamieH** [comment]
-1, it's gonna fuck shit up.

closing

**Pull request closed without merging**